### PR TITLE
Note dialog title enhancement

### LINKED ISSUE #94: Note dialog title improvements
Background color should be transparent and change to `getFormColor()` on hover and focus.

### PR DESCRIPTION
**Pull request by wkud**
Fixes #94

### CODE REVIEW

File: src/components/pages/Notes/NoteDialog/NoteDialogForm.js [hunk 1]
```diff
@@ -1,11 +1,13 @@
...
+import React, { useState } from 'react';
 import { Modal, Form } from 'react-bootstrap';
 import Todo from '../Todo';
 import { getVariant, getFormColor } from '../../../../utility/colorUtility';
 
 const NoteDialogForm = props => {
   const { children, onSubmit, formData, setFormData } = props;
   const { title, content, color } = formData;
+  const [hover, setHover] = useState();
+  const [focus, setFocus] = useState(false);
```
Comment (c1rcle) [inline src/components/pages/Notes/NoteDialog/NoteDialogForm.js]: These two states could be in a single object.

File: src/components/pages/Notes/NoteDialog/NoteDialogForm.js [hunk 2]
```diff
@@ -29,7 +31,14 @@ const NoteDialogForm = props => {
             placeholder='Note Title'
             tabIndex='1'
             className={`text-${getVariant(color)} placeholder-${getVariant(color)}`}
-            style={{ backgroundColor: getFormColor(color), borderWidth: '0' }}
+            style={{
```
Comment (c1rcle) [inline src/components/pages/Notes/NoteDialog/NoteDialogForm.js:35]: The title should be bold.

Comment (wkud) [reply src/components/pages/Notes/NoteDialog/NoteDialogForm.js:35]: done

Comment (wkud) [reply src/components/pages/Notes/NoteDialog/NoteDialogForm.js:35]: done

File: src/components/pages/Notes/NoteDialog/NoteDialogForm.js [hunk 3]
```diff
@@ -28,7 +30,15 @@ const NoteDialogForm = props => {
...
+              backgroundColor: titleState.hover || titleState.focus ? getFormColor(color) : color,
+              borderWidth: '0',
+              fontWeight: 'bold'
```
Comment (c1rcle) [inline src/components/pages/Notes/NoteDialog/NoteDialogForm.js]: Static styles should be defined as a class in a separate `css` file.

**Review by c1rcle — CHANGES_REQUESTED** [review]

**Review by c1rcle — APPROVED** [review]

**Pull request merged**